fetch.bs

```diff
@@ -1800,6 +1805,15 @@ initially unset.
 being provided to an API that didn't make a range request. See the flag's usage for a detailed
 description of the attack.
 
+<p>A <a for=/>response</a> has an associated
+<dfn for=response id=concept-response-timing-allow-failed-flag>timing allow check flag</dfn>, which
+is initially unset.
+
+<p class=note>This is used so that the caller to a fetch can determine if sensitive timing data is
```

Comment:
Interesting! Did we have requests to expose TAO availability to JS? Or is it just a "let's expose it because we can and it's cheap"?

Reply:
IIUC this is not exposed to JS - it's a concept field.

Reply:
oh, sorry. Misunderstood the note (jetlag...)